src/docs/product/integrations/integration-platform/ui-components/index.mdx

```diff
@@ -41,3 +41,5 @@ For example, if the webhook URL is `https://example.com/webhook/`, and your sche
 ## Error Handling
 
 Component rendering either 100% works or shows nothing. To protect the integration from looking chaotic due to errors we have no control over, if any part of the third-party component rendering fails, nothing will render.
+
+If there is an error in an issue linking component the link will be disabled.
```

Comment:
I am trying to not be overly wordy here - stack trace linking and alert rule UI components will not be rendered at all if there are errors, but issue links will show as disabled. Maybe I can reword:

> To protect the integration from looking chaotic due to errors we have no control over, if any part of the third-party stack trace linking or alert rule UI component rendering fails, nothing will render.  If there is an error in an issue linking component the link will be rendered as disabled.